### PR TITLE
Use "full" localization ids in the `ColorPicker` class (PR 18674 follow-up)

### PR DESCRIPTION
**Pull request by Snuffleupagus**
Apparently I missed these in PR #18674.

### CODE REVIEW

File: src/display/editor/color_picker.js [hunk 1]
```diff
@@ -81,6 +83,14 @@ class ColorPicker {
       editor?.color ||
       this.#uiManager?.highlightColors.values().next().value ||
       "#FFFF98";
+
+    ColorPicker.#l10nColor ||= Object.freeze({
```
Comment (calixteman) [inline src/display/editor/color_picker.js:87]: Why not a map ?

**Review by calixteman — APPROVED** [review]
LGTM. Thank you.

File: src/display/editor/color_picker.js (hunk 1 above)
Comment (Snuffleupagus) [reply src/display/editor/color_picker.js:87]: It seemed nice to have a structure that cannot be accidentally modified elsewhere in the code.

**Pull request merged**